server/src/server.js

```diff
@@ -767,9 +726,9 @@ app.post("/api/set-title/:id/:domain", function (req, res) {
     simpleResponse(res, "Not logged in", 401);
     return;
   }
-  Shot.get(req.backend, shotId).then((shot) => {
-    if (shot.deviceId !== req.deviceId) {
-      simpleResponse(res, "Not the owner", 403);
+  Shot.get(req.backend, shotId, req.deviceId).then((shot) => {
```

Comment:
So since req.deviceId is getting passed in to get now, if the user doesn't own the shot, they just will get a 404 instead of 403 Not the owner, right?